notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1345_TEA_Vibration_audio_freq_analysis.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    n= str('2024-03-10_13.13.06') # event instances mentioned above number of the audio file.  
</pre><p>Be careful, the audio files are stereo recording, you need to extract both channels, you can use this online tool to do that: <a href="https://audioalter.com/downmixer" target="_blank">https://audioalter.com/downmixer</a></p><p>By keeping both audio channel, I guess that you are doubling the samples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I was not aware of this (never worked recorded audio files before )- Thanks for the tip!